test/salt/pillar/debian.sls

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
It looks like the `debian`, `redhat` and `suse` only have minor differences between them (`IdentLookups` missing on suse, `User` and `Group` different on all), it could be interesting to see if a single pillar file would be possible. Or if it's interesting to have a `common` pillar file and only set specifics in those OS-named files.

Reply:
@daks Good catch, so after separating the common values into perhaps a `default.sls`, you'll end up with something like the following.

`debian.sls`:

```yaml
---
proftpd:
  IdentLookups: '"off"'
  User: nobody
  Group: nogroup
```

`redhat.sls`:

```yaml
---
proftpd:
  IdentLookups: '"off"'
  User: nobody
  Group: nobody
```

`suse.sls`:

```yaml
---
proftpd:
  User: ftp
  Group: ftp
```

Now if you _really_ wanted to get crazy, you could leave everything in `default.sls` and _only_ override for `suse`, something like:

`suse.sls`:

```yaml
---
proftpd:
  IdentLookups: ''
  User: ftp
  Group: ftp
```